components/_patterns/04-templates/example-page/example-page.twig

```diff
@@ -1,11 +1,17 @@
 <div class="example-page">
 {% include "@atoms/02-text/00-headings/h1/h1.twig" with {
-  "heading": "This is an Example Page"
-}%}
+  "heading": "This is an Example Page",
+  "h1_modifiers": {
```

Comment:
I don't know why I didn't mention this before, but this can be written shorter, like this:
```
"h1_modifiers": ["margin"],
```

And I'll give another example of multiples below